tests/serde.rs

```diff
@@ -169,8 +180,40 @@ fn numbers() {
         to_value(&std::u64::MAX).unwrap_err();
     }
 
-    test_float!(f32);
-    test_float!(f64);
+    let big_int_serializer = Serializer::new().serialize_64_bit_numbers_as_big_int(true);
+
+    {
+        const MAX_SAFE_INTEGER: i64 = 9_007_199_254_740_991;
+
+        // u64 and i64 should serialize the same
+        test_via_into_with_config(0_i64, 0_u64, &big_int_serializer);
+        test_via_into_with_config(42_i64, 42_u64, &big_int_serializer);
+
+        // Js-numbers should also deserialize into 64 bit types
+        from_value::<i64>(JsValue::from_f64(1.0)).unwrap();
```

Comment:
This only calls `from_value`, but doesn't verify the result - I think `assert_eq!` should be there for tests.

Also worth adding a test that verifies that invalid floating numbers fail (e.g. `1.5` to `i64` or `-10.1` to `u64`).

Reply:
Good Point